app/controllers/collections_controller.rb

```diff
@@ -79,10 +79,26 @@ def wait
     redirect_to collection_path(@collection) unless @collection.deposit_registering_or_updating?
   end
 
-  def works
+  def works # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     authorize! @collection
 
-    @works = authorized_scope(@collection.works, as: :collection, scope_options: { collection: @collection })
+    collection_works_result = @collection.works
+    @search_term = params[:q]
+    if @search_term
+      collection_works_result = collection_works_result.joins(
+        :user
+      ).where('title ILIKE ?', "%#{@search_term}%").or(
+        Work.where('druid ILIKE ?', "%#{@search_term}%")
+      ).or(
+        User.where('email_address ILIKE ?', "%#{@search_term}%")
+      ).or(
+        User.where('name ILIKE ?', "%#{@search_term}%")
+      ).or(
+        User.where('first_name ILIKE ?', "%#{@search_term}%")
+      )
+    end
```

Comment:
happy to factor out the updated query logic into a helper method, if people would prefer that to disabling the rubocop complaint.  i found this readable, but could've gone either way.